test/functional/test-standard-actions.js

```diff
@@ -354,6 +354,57 @@ describes.sandboxed('StandardActions', {}, () => {
     });
   });
 
+  describes.realWin('#whitelist', {
```

Comment:
I think `#foo` is usually for describing a method in the class under test. In this case, `whitelist` is not really a method, so we probably want to describe the behavior in English (i.e., drop the `#`).